vm/error_test.go

```diff
@@ -0,0 +1,35 @@
+package vm
+
+import "testing"
+
+func TestUndefinedMethodError(t *testing.T) {
+	evaluated := testEval(t, "a")
+	_, ok := evaluated.(*UndefinedMethodErrorObject)
+	if !ok {
+		t.Errorf("Expect UndefinedMethodError. got=%T (%+v)", evaluated, evaluated)
+	}
+}
+
+func TestUndefinedMethodErrorStopsProgram(t *testing.T) {
+	evaluated := testEval(t, `a; 1 + 1`)
+	_, ok := evaluated.(*UndefinedMethodErrorObject)
+	if !ok {
```

Comment:
If you do what I said above, you can change this into
```go
errObj, ok := evaluated.(*Error)
if !ok {
  // Expect object to be error object
}

if errObj.class.ReturnName != UndefinedMethodError // This is constant {
    // And this can be wrapped as a function
    t.Errorf("Expect %s. got=%T (%+v)", UndefinedMethodError, evaluated, evaluated)
}